ui/src/components/Footer/index.tsx

```diff
@@ -33,6 +33,13 @@ const Index = () => {
     <footer className="bg-light">
       <Container className="py-3">
         <p className="text-center mb-0 small text-secondary">
+          <a className="m-2" href="/privacy" target="_blank">
+            Privacy Policy
+          </a>
+          <a href="/tos" target="_blank">
+            Terms of Service
```

Comment:
Content should be translated

Reply:
if u have the transaletd privacy policy and terms of service it will surely work out

Reply:
> if u have the transaletd privacy policy and terms of service it will surely work out

What I mean is that 'Terms of Services' should use i18n translation, not the content inside.

eg：
```
// import useTranslation
import { Trans, useTranslation } from 'react-i18next';

// Add translated content in tooter in `i18m/en_US.yaml`
footer:
    build_on: >-
      Powered by <1> Apache Answer </1>- the open-source software that powers Q&A
      communities.<br />Made with love © {{cc}}.
    privacy: Privacy Policy
   terms_of_service: 

// Use where translation is required on the page
const { t } = useTranslation('translation', { keyPrefix: 'footer' });

<a className="me-2" href="/privacy" target="_blank">
            {t('privacy')}
          </a>
          <a href="/tos" target="_blank">
            {t('terms_of_service')}
          </a>

```